runtime-sdk/src/modules/core/mod.rs

```diff
@@ -83,6 +83,10 @@ pub enum Error {
     #[error("multisig too many signers")]
     #[sdk_error(code = 16)]
     MultisigTooManySigners,
+
+    #[error("invariants check failed: {0}")]
+    #[sdk_error(code = 17)]
+    InvChkFailed(&'static str),
```

Comment:
```suggestion
    #[error("invariant violation: {0}")]
    #[sdk_error(code = 17)]
    InvariantViolation(String),
```
(Use `String` so it is possible to use something like `format!`.)